### DISCUSSION TITLE
How to customize the python path instead use the system default python?

### DISCUSSION THREAD

**Discussion by OriLiMu** [qutebrowser/qutebrowser#5774] [Q&A] [answered]
I use anaconda version python. the config.py is executed by the default python in system. How to change it.
I do some search but can't find the solution

**Comment by The-Compiler** [comment, marked as answer]
`config.py` is run with whatever Python you run qutebrowser with. What that is depends entirely on how your system is set up and how you installed qutebrowser.

However, I wouldn't recommend running qutebrowser with Anaconda's Python, so far it only caused trouble because their Qt packaging seems to be quite broken.

**Comment by SohamB12** [comment]
If you want to change the folder where you save the the .py files, then when you install python use python idle instead of python command line and go to files, save as, browse and to the folder you want to save

**Reply by The-Compiler** [reply]
This seems entirely unrelated to both qutebrowser and the question being asked.

**Comment by SohamB12** [comment]
So what did u want to know
